src/app/Main/Body/QueryPage/Parameter/QueryPage-ParameterContent.component.tsx

```diff
@@ -45,12 +45,12 @@ export function QueryPageParameterContentComponent(): JSX.Element {
         }
     });
 
-    const AddParameterElement = ((filterAttributes: FilterType[], filterValues: FilterInterface): JSX.Element[] => {
+    const createParameterComponent = (
```

Comment:
actually you create component[s]

Reply:
I will make a separate commit in the refactoring only for renaming variables. :-D